functional_tests/scylla_operator/test_functional.py

```diff
@@ -581,3 +581,13 @@ def test_scylla_yaml_override(db_cluster, scylla_yaml):  # pylint: disable=too-m
 
     with scylla_yaml() as props:
         assert configmap_scylla_yaml_content == props
+
+
+@pytest.mark.readonly
+def test_default_dns_policy(db_cluster: ScyllaPodCluster):
+    pods = db_cluster.k8s_cluster.kubectl("get pods -l scylla/cluster=sct-cluster -o yaml", namespace=SCYLLA_NAMESPACE)
```

Comment:
"sct-cluster" should not be used directly. It is just default value of the "k8s_scylla_cluster_name" config option. Use config options only to get the data defined by it.

Reply:
done